### ISSUE TITLE
Pax ini file breaks some text editors

### ISSUE THREAD

**Issue by coderdj** [XENON1T/pax#154]
Specifically the neural network part of the ini file. There's like 25,000 characters in a single line. When I opened it with gedit the thing had no idea what to do. Emacs of course handled it fine but the question remains what is a 25,000 character line doing in an ini file?

Also this reminds me, if we are going to use the neural network we should probably justify the structure of it at some point (number of nodes, number of hidden layers, training set size, validation set size, content of training and validation sets, etc).  But this is an analysis issue, not a pax programming issue.
